auth0/src/main/java/com/auth0/android/request/internal/Jwt.kt

```diff
@@ -58,27 +58,29 @@ internal class Jwt(rawToken: String) {
         }
     }
 
-    private fun splitToken(token: String): Array<String> {
-        var parts = token.split(".").toTypedArray()
-        if (parts.size == 2 && token.endsWith(".")) {
-            // Tokens with alg='none' have empty String as Signature.
-            parts = arrayOf(parts[0], parts[1], "")
-        }
-        if (parts.size != 3) {
-            throw IllegalArgumentException(
-                String.format(
-                    "The token was expected to have 3 parts, but got %s.",
-                    parts.size
+    companion object {
+        fun splitToken(token: String): Array<String> {
+            var parts = token.split(".").toTypedArray()
+            if (parts.size == 2 && token.endsWith(".")) {
+                // Tokens with alg='none' have empty String as Signature.
+                parts = arrayOf(parts[0], parts[1], "")
+            }
+            if (parts.size != 3) {
+                throw IllegalArgumentException(
+                    String.format(
+                        "The token was expected to have 3 parts, but got %s.",
+                        parts.size
+                    )
                 )
-            )
+            }
+            return parts
         }
-        return parts
-    }
 
-    private fun String.decodeBase64(): String {
-        val bytes: ByteArray =
-            Base64.decode(this, Base64.URL_SAFE or Base64.NO_WRAP or Base64.NO_PADDING)
-        return String(bytes, Charsets.UTF_8)
+        fun decodeBase64(encoded: String): String {
```

Comment:
@adamjmcgrath why are we making this change from an extension function to a method?

Reply:
Since I'm turning it from private to public, I thought I might pollute the String class